integration_tests/cli/auth-demo.sh

```diff
@@ -231,12 +244,25 @@ check_response_and_report "$result"  '"info": "Alice proposes to revoke account"
 
 test_divider
 
+echo "Get list revoked account must be exist"
```

Comment:
Message should be `Get all revoked accounts. No $user account in the list` to be consistent

Reply:
Done! ✅